rules/execute-command.js

```diff
@@ -91,10 +91,13 @@ function runCommand(command, args = []) {
     execOrExit(`${node} ${yarn} run ${command} ${params}`, options);
   } else if (command.includes('${NODE}')) {
     // Support `build = "${NODE} ${ROOT_DIR}/foo.js"` as a web_binary build argument (instead of a package.json script name)
+    const loaderPath = join(rootDir, '.pnp.loader.mjs');
+    const loaderArgs = exists(loaderPath) ? `` : `--loader ${loaderPath}`;
```

Comment:
should probably quote loaderPath in this bash string, in case rootDir contains a space